### ISSUE TITLE
Installer: jupyter notebook launcher tries to find non-existing anaconda environment

### ISSUE THREAD

**Issue by gabriel-happle** [architecture-building-systems/CityEnergyAnalyst#2040]
to solve the issue of not being able to launch jupyter notebook with the CEA console
```
C:\Users\Gabriel\Documents\CityEnergyAnalyst
λ jupyter notebook
Fatal error in launcher: Unable to create process using '"c:\users\darthoma\miniconda2\envs\cea\python.exe"  "C:\Users\Gabriel\Documents\CityEnergyAnalyst\Dependencies\Python\Scripts\jupyter-notebook.EXE" '
```

We suggest to `pip install --force-reinstall jupyter` .

However, this process is blocked by an error regarding `functools32`:

```
...
...
Found existing installation: jupyter-client 5.2.4
    Uninstalling jupyter-client-5.2.4:
      Successfully uninstalled jupyter-client-5.2.4
  Found existing installation: ipykernel 4.10.0
    Uninstalling ipykernel-4.10.0:
      Successfully uninstalled ipykernel-4.10.0
  Found existing installation: functools32 3.2.3.post2
ERROR: Cannot uninstall 'functools32'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall.
```

To solve this error in the folder `CityEnergyAnalyst\Dependencies\Python\Lib\site-packages` delete the folder `functools32` and the file `functools32-3.2.3_2-py2.7.egg-info`.

Then do `pip install --force-reinstall jupyter` in the CEA console.

In the future the installer should be fixed.

**Comment by gabriel-happle** [comment]
@daren-thomas is the issue accurately described?

**Comment by daren-thomas** [comment]
@gabriel-happle yes, i think that sums it up nicely.

**Comment by daren-thomas** [comment]
This is addressed in [PR #2093](https://github.com/architecture-building-systems/CityEnergyAnalyst/pull/2093) - the installer now installs jupyter and ipython (they're just too useful not to have out of the box)

Also, try this: `start jupyter qtconsole` - i like that one a lot!
